scripts/ufm_validation_test/run_validation_test.py

```diff
@@ -0,0 +1,122 @@
+        "CheckDuplicateGuids", "CheckLinkSpeed", "CheckLinkWidth", "CheckPartitionKey", "CheckTemperature",
+        "CheckCables", "CheckEffectiveBER", "CheckSymbolBER", "RailOptimizedTopologyValidation",
+        "DragonflyTopologyValidation", "SHARPFabricValidation", "TreeTopologyValidation", "SocketDirectModeReporting"
+    ]
+    API_PARAM_TEST = "test"
+
+    VALIDATION_TEST_OPERATIONS = {
+        "run_test": "run_test",
+        "get_available_tests": "get_available_tests"
+    }
+
+    args_list = [
+        {
+            "name": f'--{VALIDATION_TEST_OPERATIONS.get("run_test")}',
+            "help": "test name to be run"
+        },
+        {
+            "name": f'--{VALIDATION_TEST_OPERATIONS.get("get_available_tests")}',
+            "help": "get list of tests which are available to run",
+            "no_value": True
+        }
+    ]
+
+
+class UfmValidationTestConfigParser(ConfigParser):
+    def __init__(self, args):
+        super().__init__(args)
+
+
+class UfmValidationTest:
+
+    @staticmethod
+    def run_validation_test(test_name):
+        url = f'{UfmValidationTestConstants.VALIDATION_TEST_API_URL}/{test_name}'
+        response = ufm_rest_client.send_request(url, HTTPMethods.POST, payload=None)
+        if response and response.status_code == HTTPStatus.ACCEPTED:
+            job_id = job_polling.extract_job_id(response.text)
+            job_polling.start_polling(job_id, print_summary=True)
+        else:
+            Logger.log_message(response, LOG_LEVELS.ERROR)
+        return response
+
+
+if __name__ == "__main__":
+    # init app args
+    args = ArgsParser.parse_args("UFM Fabric Validation Test", UfmValidationTestConstants.args_list)
+
+    # init app config parser & load config files
+    config_parser = UfmValidationTestConfigParser(args)
+
+    # init logs configs
+    logs_level = config_parser.get_logs_level()
+    logs_file_name = config_parser.get_logs_file_name()
+    Logger.init_logs_config(logs_file_name, logs_level)
+
+    # init ufm rest client
+    ufm_rest_client = UfmRestClient(host=config_parser.get_ufm_host(),
+                                    client_token=config_parser.get_ufm_access_token(), username=config_parser.get_ufm_username(),
+                                    password=config_parser.get_ufm_password(), ws_protocol=config_parser.get_ufm_protocol())
+    job_polling = JobPolling(ufm_rest_client, "validation test")
+    args_dict = args.__dict__
+    if args_dict.get(UfmValidationTestConstants.VALIDATION_TEST_OPERATIONS.get("run_test")):
+        param_test_name = args_dict.get(UfmValidationTestConstants.VALIDATION_TEST_OPERATIONS.get("run_test"))
+        if param_test_name and param_test_name in UfmValidationTestConstants.AVAILABLE_VALIDATION_TESTS:
+            UfmValidationTest.run_validation_test(param_test_name)
+        else:
+            message = f'Please provide valid test name'
```

Comment:
it's better to list the available tests here:
The test <test_name> isn't one of the following available tests: [tests array]